manifests/init.pp

```diff
@@ -32,6 +32,10 @@
 #                                   Setting this to anything non-empty causes
 #                                   the apache vhost to set up a proxy for all
 #                                   certificates pointing to the value.
+# $parent_reverse_proxy::           Add reverse proxy to the parent
```

Comment:
Can this just be $reverse_proxy? Since this will be exposed as Kafo parameters, i like `--capsule-reverse-proxy=true` instead of `--capsule-parent-reverse-proxy=true`.  It also needs to match the name of the feature being enabled, if we will some day have some automagic Smart Proxy + Foreman plugins...

Speaking of those, what are these? @dustints?

https://github.com/Katello/foreman_reverse_proxy
https://github.com/Katello/smart-proxy-http-reverse-proxy-plugin
https://github.com/Katello/smart-proxy-templates-plugin
https://github.com/Katello/smart-proxy-rhsm-plugin

smart-proxy-http-reverse-proxy-plugin looks the most active, but I don't follow the name.  Foreman's Smart Proxy plugin names correlate directly to the Feature name.  This should be `smart_proxy_reverse_proxy` so the feature becomes "Reverse Proxy" in the Foreman UI.